synapse/tests/test_lib_types.py

```diff
@@ -367,6 +369,38 @@ def test_range(self):
         self.raises(s_exc.BadTypeDef, model.type('range').clone, {'type': None})
         self.raises(s_exc.BadTypeDef, model.type('range').clone, {'type': ('inet:ipv4', {})})  # inet is not loaded yet
 
+    async def test_range_filter(self):
+
+        async with self.getTestCore() as core:
+
+            async with await core.snap() as snap:
+                node = await snap.addNode('inet:ipv4', '3.3.3.3')
+                node = await snap.addNode('inet:ipv4', '172.111.92.6')
+                node = await snap.addNode('inet:ipv4', '4.4.4.5')
+                node = await snap.addNode('inet:ipv4', '192.168.1.1')
+                node = await snap.addNode('file:bytes', b'f')
+                node = await snap.addNode('file:bytes', b'foo')
+                node = await snap.addNode('file:bytes', b'ffoo')
+                node = await snap.addNode('file:bytes', b'foobar')
+                node = await snap.addNode('file:bytes', b'A Slightly longer string for a test')
```

Comment:
Since the `*range=` helper was added to the base `Type` class, we can have range filter operations which do not operate on things which are cleanly integer valued.  The lack of coverage here for various other types (especially ndef based comparisons on Edge types which may have different lengths or Python object types) could lead to undefined behaviors occurring.  We should definitely include a more expansive set of base types being tested here.  One failsafe to fall back to is on a `TypeError` is to catch and return False.